converter/grpc_interceptor_test.go

```diff
@@ -74,6 +74,7 @@ func TestPayloadCodecGRPCClientInterceptor(t *testing.T) {
 	)
 	require.NoError(err)
 
+	//lint:ignore SA1019 test
```

Comment:
Is this to ignore deprecation? Which thing here is deprecated?

Reply:
Looks like they did in 1.63.0 and just changed their mind in 1.63.1, putting it off until 1.64.0: https://github.com/grpc/grpc-go/pull/7103.

Reply:
Yeah but they still plan to deprecate it